schema.graphql

```diff
@@ -42,6 +42,10 @@ type Token @entity {
   priceUsd: BigDecimal
   "History of trading for this token"
   history: [TokenTradingEvent!]! @derivedFrom(field: "token")
+  "Hourly totals"
+  hourlyTotals: [TokenDailyTotal!]! @derivedFrom(field: "token")
+  "Daily totals"
+  dailyTotals: [TokenDailyTotal!]! @derivedFrom(field: "token")
```

Comment:
Is this an unrelared change?

Reply:
Also, both hourly and daily values are assigned TokenDailyTotal

Reply:
My bad! already solved in other commit and merged branch